index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <h1>Hello microverse</h1>
```

Comment:
- [ ] I think it's better to edit your README using the [Microverse Template](https://github.com/microverseinc/readme-template) in this branch, then commit and push those changes, because it is very important to document your projects in an Industry-standard convention and **before this we cannot approve your project**

|Your README| The Template You should use|
| ---- | ---- |
| ![readme](https://user-images.githubusercontent.com/102636489/228146356-18d47791-09b4-4b51-91d1-863541fb35a7.png) | ![readmeII](https://user-images.githubusercontent.com/102636489/228146435-a3ae6fbb-e91f-45ad-90f1-7ff784922a92.png) |